src/main/java/com/sourcegraph/webhook/Dispatcher.java

```diff
@@ -2,7 +2,7 @@
 
 import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
 import com.atlassian.sal.api.net.Request;
-import com.atlassian.sal.api.net.RequestFactory;
+import com.atlassian.sal.api.net.NonMarshallingRequestFactory;
```

Comment:
sal no longer exposes an instance of `RequestFactory`, but it exposes `NonMarshallingRequestFactory` which works just as fine for our use-case as far as I can tell. This was causing the plugin to silently timeout when enabling.